linode/manager
# refactor: Un-Redux-ify Object Storage Drawer

## Description 📝

- Removes Object Storage Bucket Create Drawer form Redux because it created unnecessary complexity in a simple flow

## How to test 🧪

- Test Creating an Object Storage Bucket
- Test deep linking of `http://localhost:3000/object-storage/buckets/create` (drawer should be open)
- Test enabling object storage for the first time (I used the MSW to mock this)
